alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,65 @@
+# AggregatedAPIErrors
+
+## Meaning
+
+This alert is triggered in case multiple calls on the aggregated API of OpenShift failed over a certain time window.
+
+## Impact
+Errors on the aggregated API can result in the unavailability of some OpenShift services.  
+
+## Diagnosis
+
+##### Check the APIs status checks are on True
+
+Currently, there are three aggregated APIs in an OpenShift Cluster. The API on the `openshift-apiserver` namespace, the prometheus-adapter on the namespace `openshift-monitoring`, and the package-server service in the `openshift-operator-lifecycle-manager` namespace.
```

Comment:
Can't admins install additional aggregated APIs?  Maybe "Currently, there are at least three..."?

Reply:
Ah, I see you are using grep to exclude aggregated APIs which may be living in other namespaces.  But the `oc get apiservice` call looks pretty clear to me; maybe we can just drop this paragraph and let the output speak for itself?  There's always `oc explain ...` for folks who want more context, without us having to inline a summary here.

Reply:
This part is just a copy from the private runbooks. But should we exclude the other namespaces? 
The same error can occur for all additionally created aggregated APIs, right? See https://github.com/openshift/runbooks/pull/1#discussion_r600338635

Reply:
yeah, I don't think we want to exclude other namespaces.